resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -54,6 +54,12 @@ private[spark] object Config extends Logging {
       .checkValues(Set("Always", "Never", "IfNotPresent"))
       .createWithDefault("IfNotPresent")
 
+  val IMAGE_PULL_SECRET =
+    ConfigBuilder("spark.kubernetes.imagePullSecret")
```

Comment:
Given there's an option `spark.kubernetes.container.image.pullPolicy`, we should make this consistent as `spark.kubernetes.container.image.pullSecret`

Reply:
Done